docs/source/faq.md

```diff
@@ -115,6 +115,33 @@ It allows customization of the various parameters. An environment variable
 has precedence over the value defined in `ucx.conf`.
 The file can be created using `ucx_info -Cf`.
 
+#### Build user application with UCX
+
+UCX supports metainformation subsystem based on `pkg-config` tool. To add UCX support
+into application, output from `pkg-config` tool may be used in Makefile:
+```
+program: program.c
+        cc program.c $(pkg-config --cflags --libs ucx)
+```
+To build static application with UCX supporting just specific network transport
+modules (e.g. CMA and KNEM), the following command can be used:
+```
+program: program.c
+        cc -static program.c $(pkg-config --cflags --libs --static ucx-cma ucx-knem ucx)
+```
+Currently supported modules are: `ucx-cma`, `ucx-ib`, `ucx-knem`, `ucx-xpmem` and `ucx-rdmacm`.
+Transports `TCP`, `SHM` and `self` are built-in into UCT and don't need additional
+compilation actions.
```

Comment:
TCP, basic shared memory, and self transports are built-in into UCT and don't need additional
compilation actions.

Reply:
reworded